src/mrinufft/operators/interfaces/finufft.py

```diff
@@ -40,6 +40,8 @@ def __init__(
             self._make_plan(i, **kwargs)
             self._set_pts(i)
 
+        self._make_plan_grad(**kwargs)
+
```

Comment:
```suggestion
```

Same as above. We dont need additional plan which will take up more memory. What we need to do is to trigger this creation only when `grad_wrt_traj=True`